tests/preprocessing/test_normalization.py

```diff
@@ -87,12 +89,35 @@ def test_vars_checks(adata_to_norm):
         ep.pp.scale_norm(adata_to_norm, vars=["String1"])
 
 
-@pytest.mark.parametrize("array_type", ARRAY_TYPES)
-def test_norm_scale(array_type, adata_to_norm):
+# TODO: list the supported array types centrally?
+norm_scale_supported_types = [np.asarray, da.asarray]
+norm_scale_unsupported_types = [sparse.csc_matrix]
+
+
+# TODO: check this for each function, with just default settings?
+@pytest.mark.parametrize(
```

Comment:
I picked strict three array types to be tested rigidly: `np.ndarray`, `dask.array.Array`, `scipy.sparse.spmatrix`. Not sure if we should test for more. I personally favor being strict about these three things, and not guaranteeing anything else.

Reply:
Yeah totally fine. cupy arrays are for the future